docs/docs/guides/getting_started/introduction.md

```diff
@@ -21,6 +21,9 @@ The following documentation will guide you through different use cases of Web3.j
 -    More efficient ABI Encoder & Decoder
 -    Custom Output formatters
 -    In compliance with Eth EL API
+-    Reduced build size of packages
+
+![Package Size Chart](PackageSizeChart.png)
```

Comment:
@Deepak2030 we need comparison of web3 packages with other libs, 
**potentially research and add other advantages of web3.js over other libraries and add them**

@SantiagoDevRel web3 package size is changed with time so we will need to keep this updated!

Reply:
I've updated the PR, let me know if it needs any improvement